modules/zookeeper/pom.xml

```diff
@@ -40,6 +40,12 @@
             <version>${project.version}</version>
         </dependency>
 
+        <dependency>
```

Comment:
Please revert this. This changes doesn't related with your ticket

Reply:
Reverted